argilla/docs/getting_started/huggingface-spaces.md

```diff
@@ -0,0 +1,147 @@
+
+
+## Enable persistent storage
+
+!!! warning
+    Activate persistent storage not to loose your data. If you are not using a paid Space upgrade, any Space will be shut down after 48 hours of inactivity. When restarted, it will undergo a factory reset, resulting in the loss of all your data.
+
+    Make sure to enable persistent storage before adding any data, as the Space will restart, removing all the previous data.
+
+To enable [persistent storage](https://huggingface.co/docs/hub/spaces-storage#persistent-storage), go to the "Settings" tab on your created Space and click on the desired plan on the "Persistent Storage" section.
+
+If you haven’t enabled persistent storage, Argilla will display a warning message by default. To prevent this warning from appearing if you don’t need persistent storage, set the environment variable `ARGILLA_SHOW_HUGGINGFACE_SPACE_PERSISTENT_STORAGE_WARNING` to `false` in "Settings" > "New variable". This will suppress the warning message.
+
+## Configure the Space secrets
+
+[Secrets](https://huggingface.co/docs/hub/spaces-overview#managing-secrets) can be configured as optional settings to secure your Argilla Space.
+
+The Argilla template allows you to configure the credentials for the default users: `owner`, `admin` and `annotator`. It will also allow you to configure the name of the default workspace.
+
+> Check this [guide](../how_to_guides/user.md) to learn more about users, and refer to this [guide](../how_to_guides/workspace.md) for more information about workspaces in Argilla.
+
+Moreover, it provides two secrets to configure the Hugging Face Oauth. For more information, check the [next section](#sign-in-with-hugging-face).
+
+You can add the available secrets in "Settings" > "New secret".
+
+??? note "Available secrets"
+
+    | Secret name     | Type | Description | Default value   |
+    |-------------------------------|----------|-------------|----------------|
+    | `OWNER_USERNAME` | string | A username to log in with owner permissions | owner |
+    | `OWNER_PASSWORD` | string | A password for the `OWNER_USERNAME` | 12345678 |
+    | `OWNER_API_KEY` | string or online generator |An `api_key` to interact with the SDK with owner permissions | owner.apikey |
+    | `ADMIN_USERNAME` | string |A username to log in with admin permissions | admin |
+    | `ADMIN_PASSWORD` | string | A password for the `ADMIN_USERNAME` | 12345678 |
+    | `ADMIN_API_KEY` | string or online generator|An `api_key` to interact with the SDK with admin permissions | admin.apikey |
+    | `ANNOTATOR_USERNAME` | string |A username to log in with annotator permissions | argilla |
+    | `ANNOTATOR_PASSWORD` | string |A password for the `ANNOTATOR_PASSWORD` | 12345678 |
+    | `ARGILLA_WORKSPACE` | string | The name of the default workspace | admin |
```

Comment:
persistent storage one missing.

Reply:
I didn't add it because it was an env var, not needed as a secret, but I can add it